<launch>

  <arg name="robot_pose_x" value="-5.0"/>
  <arg name="robot_pose_y" value="0.0"/>
  <arg name="robot_pose_Y" value="1.0"/>

  <param name="robot_description" command="$(find xacro)/xacro.py $(find turtlebot3_description)/urdf/turtlebot3_waffle.urdf.xacro"/>
  <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
  <node pkg="robot_state_publisher" type="robot_state_publisher" name="state_publisher"/>

  <node pkg="shared_control" type="fake_robot_vel.py" name="fake_robot_vel" output="screen">
    <param name="pose_x" value="$(arg robot_pose_x)"/>
    <param name="pose_y" value="$(arg robot_pose_y)"/>
    <param name="pose_Y" value="$(arg robot_pose_Y)"/>
    <param name="velocity_lin" value="0.26"/>
    <param name="velocity_ang" value="1.82"/>
    <param name="sim_cycle" value="0.1"/>
  </node>
  <include file="$(find shared_control)/launch/shared_control.launch"/>

  <include file="$(find shared_control)/launch/joystick.launch"/>
  <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find shared_control)/rviz/simple_test.rviz"/>

</launch>
